app/renderer/components/main/main.js

```diff
@@ -580,6 +585,33 @@ class Main extends React.Component {
     return props
   }
 
+  onDeprecationButtonClick = () => {
+    if (!this.props.braveCoreInstalled) {
+      appActions.maybeCreateTabRequested({
```

Comment:
this was re-worked into `appActions.createTabRequested` (which is confusing, because the shields reference above was never updated! whoops)